easybuild/easyconfigs/a/AutoDock/AutoDock-4.2.6-GCCcore-8.3.0.eb

```diff
@@ -0,0 +1,30 @@
+# Currently there is not an EasyBlock to unify AutoDock and AutoGrid so this is a happy medium until then
+easyblock = 'ConfigureMake'
+
+name = 'AutoDock'
+version = '4.2.6'
+
+homepage = 'https://autodock.scripps.edu/'
+
+description = """
+ AutoDock is a suite of automated docking tools. It is designed to 
+ predict how small molecules, such as substrates or drug candidates, bind to 
+ a receptor of known 3D structure.
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '8.3.0'}
+
+source_urls = [' https://autodock.scripps.edu/downloads/autodock-registration/tars/dist426/']
```

Comment:
@JackPerdue The extra space here in front is causing trouble (try `eb --from-pr 9678 --force-download --sourcepath /tmp/$USER/pr9678`)